src/components/PeoplePage.tsx

```diff
@@ -1,33 +1,113 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useSearchParams } from 'react-router-dom';
 
 export const PeoplePage = () => {
-  return (
-    <>
-      <h1 className="title">People Page</h1>
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
 
-      <div className="block">
-        <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+  const [searchParams] = useSearchParams();
+
+  const currentSort = searchParams.get('sort');
+  const currentOrder = searchParams.get('order');
+
+  const filterPeople = () => {
+    let visiblePeople = people;
+
+    const filterSex = searchParams.get('sex');
+    const filterQuery = searchParams.get('query')?.toLowerCase() || '';
+    const filterCentury = searchParams.getAll('century') || [];
```

Comment:
This function shouldn't know anything about `searchParams`, just gets people and parameters. And return a new filtered array